Portfolio/src/views/WorkView.vue

```diff
@@ -1,7 +1,34 @@
+<script setup lang="ts">
+import Wrapper from '@/Wrapper.vue';
+import { ref } from 'vue';
+
+const projects = ref([
+    { id: 1, label: "Project Name", text: "Year" },
+    { id: 2, label: "Project Name", text: "Year" },
+    { id: 3, label: "Project Name", text: "Year" }
+])
+</script>
+
 <template>
-    <div class="work">
-        <h1>This is a work page</h1>
-    </div>
+    <Wrapper>
+        <main class="work-page-wrapper">
+            <div class="work-page-text-container">
+                <p class="work-page-title">Work</p>
```

Comment:
How come a title is just a paragraph element? You are not planning to have a heading in this page? How will u structure your elements semantically?

Reply:
It's a heading, just changed it :)